test/acceptance/js/helpers/DocstoreClient.js

```diff
@@ -60,6 +60,20 @@ module.exports = DocstoreClient = {
     )
   },
 
+  peekDoc(project_id, doc_id, qs, callback) {
+    if (callback == null) {
+      callback = function (error, res, body) {}
+    }
```

Comment:
Let's not provide a default callback. We'll have to remove it when converting to async/await anyway